riptide-spring-boot-autoconfigure/README.md

```diff
@@ -606,10 +606,17 @@ The following table shows all beans with their respective name (for the `example
 | `exampleFaultClassifier`          | `FaultClassifier`                             |
 | `exampleCircuitBreakerListener`   | `CircuitBreakerListener`                      |
 | `exampleAuthorizationProvider`    | `AuthorizationProvider`                       |
+| `exampleRetryPolicyExecutorService`    | `ExecutorService`                       |
```

Comment:
It would be nice to add these beans to the beans diagram as well https://github.com/zalando/riptide/pull/1503/files#diff-cac26b0853dfe7f7df5898467d9a7d21fe3c0d656f8debcaedab57f4c6d20f39L567 - do we have an editable version?

Reply:
I assume, you are also using IntelliJ… Can you use its feature to align the table so it is properly aligned?

Reply:
Fixed